test/common/testlib.py

```diff
@@ -1314,6 +1314,17 @@ def filterOutput(self, test, failed, output):
         return failed, b"# RETRY " in output
 
 
+def print_tests(tests):
+    for test in tests:
+        if isinstance(test, unittest.TestSuite):
+            print_tests(test)
+        elif isinstance(test, unittest.loader._FailedTest):
+            name = test.id().replace("unittest.loader._FailedTest.", "")
+            print(f"Error: '{name}' does not match a test", file=sys.stderr)
```

Comment:
Let's please avoid f-strings, we can't count on a that recent Python 3 yet. (See e. g. the Semaphore failure).

Reply:
Right, removed the f-string